grails-app/conf/UrlMappings.groovy

```diff
@@ -1,10 +1,9 @@
-import grails.util.Environment
-
 class UrlMappings {
 
     static mappings = {
 
         "/proxy"(controller: "proxy", action: "index")
+        "/sitemap/**"(controller: "sitemap", action: "index")
 
```

Comment:
This is not related to this PR, but do you think it will be better to do the redirection here? i.e. search/uuid/something to search?uuid=something&info=true